mqt/benchviewer/main.py

```diff
@@ -123,9 +148,38 @@ def get_num_benchmarks():
         return jsonify(data)
 
 
-def main():
-    app.run(debug=True)
+def start_server(
+    skip_question: bool = False,
+    activate_logging: bool = False,
+    target_location: str = None,
+    debug_flag: bool = False,
+):
+
+    if not target_location:
+        package_path = importlib.util.find_spec("mqt.bench").origin
+        target_location = (
+            package_path.split("bench/__init__.py")[0] + "benchviewer/static/files/"
+        )
```

Comment:
this does not seem like the latest best practise for managing package resources. This should probably use `importlib.resources` (or the respective back port `importlib_resources` for older Python versions).
Then, this part of the code should look something like this:

```
if sys.version_info < (3, 10, 0):
    import importlib_resources as resources
else:
    from importlib import resources

target_location = resources.files(mqt.bench.benchviewer) / "static" / "files"
```

Note that `target_location` now is a `pathlib.Path` object.

Furthermore `importlib_resources` has to be added to the project dependencies for Python versions < 3.9, i.e.
```
"importlib_resources>=5.9; python_version < '3.10'"
```

Reply:
done